packages/frontend-v2/utils/theme/index.ts

```diff
@@ -49,15 +49,13 @@ const colors = {
   },
   neutral: {
     main: "#e7ebf1",
```

Comment:
[nit] maybe we can condense the 2 instances of #e7ebf1 into one instance in this file?

Reply:
so true i missed that